spec/statesman/adapters/active_record_spec.rb

```diff
@@ -308,9 +308,9 @@
       described_class.new(MyActiveRecordModelTransition, model, observer)
     end
 
-    before { adapter.create(:x, :y) }
-
```

Comment:
In tests of the `#last` method we were always creating a transition. This is not how it happens in reality.